Unithereum.CodeGen/UnithereumCodeGenProcessor.cs

```diff
@@ -36,34 +25,27 @@ public void OnPreprocessAsset()
                 Path.GetDirectoryName(Application.dataPath)!,
                 this.assetPath
             );
-            if (changedPath.EndsWith(".abi", StringComparison.OrdinalIgnoreCase))
-            {
-                var binPath = Path.ChangeExtension(changedPath, ".bin");
-                Generate(changedPath, File.Exists(binPath) ? binPath : null);
-            }
-            else if (changedPath.EndsWith(".bin", StringComparison.OrdinalIgnoreCase))
-            {
-                var abiPath = Path.ChangeExtension(changedPath, ".abi");
-                if (File.Exists(abiPath))
-                    Generate(abiPath, changedPath);
-            }
+            FindAbiBinPairAndGenerate(changedPath);
         }
 
         /// <summary>
-        /// Search for all .abi and .bin files in the Asset/ directory and force import them.
+        ///     Search for all .abi and .bin files in the Asset/ directory and force import them.
         /// </summary>
-        /// <remarks>As the <see cref="OnPreprocessAsset"/> function will generate code for imported .abi files,
-        /// this function will regenerate code for all .abi files without explicitly calling <see cref="Generate"/>.
-        /// </remarks>
         [MenuItem("Unithereum/Regenerate All...")]
         public static void RegenerateAllMenu()
         {
+            var config = GetConfig();
+            if (config is null)
+                return;
```

Comment:
I think it might make sense to handle `InvalidCodeGenConfigurationException` here as well; the user might expect a graphical feedback after they click a menu, so it might be better to show error message dialogs here if there is a config error and codegen cannot continue.

Reply:
Ah, I actually meant that the user might expect a graphical dialog with error message explaining why codegen could not take place, instead of a console log. (i.e. `EditorUtility.DisplayDialog("Generation Unavailable", errorMessage, "Confirm");`)

Reply:
Oh I see.. I'll update it